### PR TITLE
Merge canary changes to master

### PR DESCRIPTION
**Pull request by abdullai-t**
#### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [ ] I've given my PR a meaningful title.
* [ ] I linked this PR to the relevant issue.
* [ ] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [ ] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

File: src/_main_/config/build/deployConfig.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "IS_PROD": true,
-  "IS_CANARY": false,
-  "BUILD_VERSION": "5.1.10",
+  "IS_PROD": false,
+  "IS_CANARY": true,
+  "BUILD_VERSION": "5.1.17",
   "BUILD_VERSION_NOTES": "Jan 31, 2024: Campaigns initial prod deployment\n"
```
Comment (Opoku-Agyemang) [inline src/_main_/config/build/deployConfig.json:5]: are you sure u wanna do this? for prod?

File: src/_main_/settings.py [hunk 2]
```diff
@@ -39,8 +39,8 @@
     RUN_CELERY_LOCALLY = True
 
 # Database selection, development DB unless one of these chosen
-IS_PROD = True
-IS_CANARY = False
+IS_PROD = False
+IS_CANARY = True
```
Comment (Opoku-Agyemang) [inline src/_main_/settings.py:43]: nope.

**Review by Opoku-Agyemang — CHANGES_REQUESTED** [review]

File: src/_main_/config/build/deployConfig.json (hunk 1 above)
Comment (abdullai-t) [reply src/_main_/config/build/deployConfig.json:5]: no.

File: src/_main_/settings.py (hunk 2 above)
Comment (abdullai-t) [reply src/_main_/settings.py:43]: This code is from the canary branch. I will change it after merge

**Review by Opoku-Agyemang — APPROVED** [review]
make sure to run `make prepare-prod`

**Pull request merged**